composer/utils/object_store/s3_object_store.py

```diff
@@ -93,6 +93,9 @@ def __init__(
         if client_config is None:
             client_config = {}
         config = Config(**client_config)
+        if 'AWS_ENDPOINT_URL' in os.environ and endpoint_url is None:
```

Comment:
is `AWS_ENDPOINT_URL` the right name? Or is `S3_ENDPOINT_URL` more accurate?

Reply:
oh good call. updated!

Reply:
also fixed in the mcli PR